src/Service/ListService.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace OpenBibIdApi\Service;
+
+class ListService extends Service implements ListServiceInterface
```

Comment:
Niet nodig om ook nog eens "implements LibraryServiceInterface" toe te voegen; Is reeds gebeurd door de Service class.